senator/utility.py

```diff
@@ -1,21 +1,29 @@
 import numpy as np
 RESOURCES = 17
 PLAYERS = 3
-OWNER_TRASHOLD = 0.35
+#OWNER_TRASHOLD = 0.35
+#OWNER_LOSS = 0.8
 FILE_PATH = "data.csv"
 
 def get_utility(previous_utility: np.ndarray, chosen_resources: list[int], 
-                votes: np.ndarray) -> np.ndarray:
+                votes: np.ndarray, 
+                is_owner: np.ndarray = np.zeros((PLAYERS, RESOURCES)),
+                owner_loss: float = 0.8, 
+                other_loss: float = 0,
+                owner_trashold: float = 2.0) -> np.ndarray: 
+    
     num_players, num_resources = previous_utility.shape
     
     current_strategy_profile = np.zeros((num_players, num_resources))
     current_strategy_profile[np.arange(num_players), chosen_resources] = 1
 
     current_utility = np.zeros((num_players, num_resources))
     
-    
-    is_owner = previous_utility >= OWNER_TRASHOLD * votes
-    current_utility = (previous_utility / 2) * is_owner
+    if owner_trashold <= 1.0:
+        is_owner = previous_utility >= owner_trashold * votes #original variant of the model, where is_owner is computed by using a trashold
+       
+    current_utility = ((previous_utility * owner_loss) * is_owner) + (previous_utility * other_loss) * (1- is_owner)
```

Comment:
Jedna hodně hnidopišská, hodně se ti rozchází kdy děláš mezery kolem čárek, kolem znamének atd. Existujou extensions do VS code, který dělají automatický formátování. Doporučuju si něco najít a nainstalovat a pak to prostě neřešit, protože se to vyřeší za tebe :D vím že asi `pylint` má nějaký formátování, pak je `black` (ten je hodně striktní), `autopep8`. Zkus se po tom kouknout, nebo @juliakrizanova jestli třeba něco nemá, ta asi používá VS code? Nebo případně Richard